getting-started/FAQ.md

```diff
@@ -286,5 +286,10 @@ apt update && apt install apparmor-utils -y && systemctl restart containerd
 
 After this, exit out of the node shell and follow the [getting-started guide](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md).
 
+If the `kubearmor-relay` pod goes into CrashLoopBackOff, apply the following patch:
+```sh
+kubectl patch deployment kubearmor-relay -n kube-system --type=json -p='[{"op": "add", "path": "/spec/template/metadata/annotations/container.apparmor.security.beta.kubernetes.io~1kubearmor-relay-server", "value": "unconfined"}]'
```

Comment:
Will kube-system reference here be an issue since, we are moving out of it?

Reply:
Fixed. It should work regardless of the namespace in which kubearmor-relay is deployed.